### PR TITLE
Documents additional configurable features.

### LINKED ISSUE #97: Document `game_config.json`, `logging_config.json`

### PR DESCRIPTION
**Pull request by flosincapite**
Fixes #97 

Also adds `logging_config.json` and `game_config.json` files with default values to the repo.

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Developer Tricks
+
+Here are a few features which are useful when manually testing the game.
+
+## Logging
+
+To enable logging, create a file called `logging_config.json` in the root `cronenbroguelike` directory (i.e., a sibling to this file. This file recognizes the following parameters:
```
Comment (abjose) [inline DEVELOPERS.md]: Should this say "modify" rather than "create" the file?

Comment (flosincapite) [reply DEVELOPERS.md]: Ah, good catch. Fixed!

File: DEVELOPERS.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Game Config
+
+To enable various gameplay tweaks, modify `game_config.json` in the root `cronenbroguelike` directory. This file recognizes the following parameters:
+
+- `num_rooms`: an integer specifying how many rooms to place in the floor.
+- `extra_commands`: a Boolean specifying whether to add various extra commands, including [cheat](cheat).
+- `random_run`: a Boolean specifying whether to let the game play itself by producing random commands.
+
+`num_rooms` can be helpful to restrict the size of randomly generated floors when testing. `extra_commands`'s primary utility lies in the [cheat](cheat) command, which allows the player to alter the player character's statistics and abilities. `random_run` is mostly for fun.
```
Comment (xmunoz) [inline DEVELOPERS.md]:
```suggestion
`num_rooms` can be helpful to restrict the size of randomly generated floors when testing. `extra_commands`'s primary utility lies in the [cheat](#cheat) command, which allows the player to alter the player character's statistics and abilities. `random_run` is mostly for fun.
```

File: DEVELOPERS.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+To enable various gameplay tweaks, modify `game_config.json` in the root `cronenbroguelike` directory. This file recognizes the following parameters:
+
+- `num_rooms`: an integer specifying how many rooms to place in the floor.
+- `extra_commands`: a Boolean specifying whether to add various extra commands, including [cheat](#cheat).
+- `random_run`: a Boolean specifying whether to let the game play itself by producing random commands.
+
+`num_rooms` can be helpful to restrict the size of randomly generated floors when testing. `extra_commands`'s primary utility lies in the [cheat](#cheat) command, which allows the player to alter the player character's statistics and abilities. `random_run` is mostly for fun.
+
+## <a id="cheat">Cheats</a>
```
Comment (xmunoz) [inline DEVELOPERS.md]:
```suggestion
## Cheats
```

File: DEVELOPERS.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Game Config
+
+To enable various gameplay tweaks, modify `game_config.json` in the root `cronenbroguelike` directory. This file recognizes the following parameters:
+
+- `num_rooms`: an integer specifying how many rooms to place in the floor.
+- `extra_commands`: a Boolean specifying whether to add various extra commands, including [cheat](#cheat).
+- `random_run`: a Boolean specifying whether to let the game play itself by producing random commands.
+
+`num_rooms` can be helpful to restrict the size of randomly generated floors when testing. `extra_commands`'s primary utility lies in the [cheat](#cheat) command, which allows the player to alter the player character's statistics and abilities. `random_run` is mostly for fun.
```
Comment (xmunoz) [inline DEVELOPERS.md]:
```suggestion
`num_rooms` can be helpful to restrict the size of randomly generated floors when testing. `extra_commands`'s primary utility lies in the [cheat](#cheats) command, which allows the player to alter the player character's statistics and abilities. `random_run` is mostly for fun.
```

**Review by xmunoz — APPROVED** [review]

**Pull request merged**